src/app/(social)/_components/ProfileSection.tsx

```
@@ -0,0 +1,40 @@
+          width={80}
+          height={80}
+          className="rounded-full border-2 border-primary object-cover object-top"
+        />
+      </div>
+
+      <div className="font-medium">{user?.nickname}</div>
+
+      <div
+        id="user-profile-description"
+        className="text-xs text-gray-400"
+        dangerouslySetInnerHTML={{ __html: user.html_contents }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)